build/Makefile

```diff
@@ -177,43 +156,6 @@ distclean : clean clean-ruby clean-dsc clean-auoms
 clean-status :
 	@$(ECHO) "========================= Performing make clean"
 
-#--------------------------------------------------------------------------------
-# Build OMI project (configured for the OMS project)
-
-$(OMI_LIBRARY_LIB_BASE) :
-	@$(ECHO) "========================= Performing Building OMI Project"
-ifeq ($(ULINUX),1)
-	cd $(OMI_ROOT); ./configure --enable-microsoft --enable-ulinux
-else
-	cd $(OMI_ROOT); ./configure --enable-microsoft
-endif
-	$(MAKE) -C $(OMI_ROOT) all
-
-#--------------------------------------------------------------------------------
-# Build DSC project (configured for the OMS project)
-#
-# Note: This expects that OMI is already built! Take care of your
-# ordering on the 'all' target!
-
-$(DSC_TARGET_DIR) : $(OMI_LIBRARY_LIB_BASE)
-ifeq ($(ULINUX),1)
-	@$(ECHO) "========================= Performing Building DSC Project"
-	cd $(DSC_DIR); ./configure --oms
-	ln -fs $(OMI_ROOT) $(DSC_DIR)/omi-1.0.8
-	cd $(OMI_ROOT); $(RMDIR) output; ln -s output_openssl_0.9.8 output
-	$(MAKE) -C $(DSC_DIR) dsc098
-	$(MAKE) -C $(DSC_DIR) dsckit098
-	cd $(OMI_ROOT); $(RMDIR) output; ln -s output_openssl_1.0.0 output
-	$(MAKE) -C $(DSC_DIR) dsc100
-	$(MAKE) -C $(DSC_DIR) dsckit100
-ifeq ($(PF_ARCH),x64)
-	cd $(OMI_ROOT); $(RMDIR) output; ln -s output_openssl_1.1.0 output
-	$(MAKE) -C $(DSC_DIR) dsc110
-	$(MAKE) -C $(DSC_DIR) dsckit110
-endif
-	cd $(OMI_ROOT); $(RMDIR) output
-endif
```

Comment:
You should also inform about these change to oms plugin partner team. 
This change will mean that omsagent will no longer build partner plugins and new changes in partner plugins will not be published until DSC starts building in a separate jenkins.

Reply:
Sure, will update all the plugin partner teams.

@NarineM , now the travis build is failing because it needs the newly added submodules in Build-OMS-Agent-for-Linux, the build should be successfull once that PR is merged.